ivy.el

```diff
@@ -1246,6 +1247,12 @@ On error (read-only), call `ivy-on-del-error-function'."
   (unless (= (point) (line-end-position))
     (delete-char arg)))
 
+(defun ivy-delete-forward-char (arg)
+  "Forward to `delete-forward-char' ARG."
+  (interactive "p")
+  (unless (= (point) (line-end-position))
+    (delete-forward-char arg)))
```

Comment:
Why not support both interactive arguments of `delete-forward-char`?

Reply:
I didn't notice it had two arguments; I've force-pushed a new version of the patch to support both arguments.